templates/endpoints_v2/EmbeddingModels.md

```diff
@@ -0,0 +1,32 @@
+# Serving Embedding Models
+
+We support serving embedding models available in HuggingFace as well as optimizing these with ONNX. Sample configs are available in the `models/embedding_models` folder.
+
+# Setting up Model
+
+See an example for serving embedding models in `serve_embedding.yaml`. Notably the serve arguments need to contain the `embedding_models` field. 
```

Comment:
I don't get the 2nd half of it? Which serve arguments? And how to add this field?